README.rst

```diff
@@ -58,6 +76,15 @@ Once installed, the plugin should automatically register itself within Django. B
 
 The admin views are in the Django admin, under the "Section to Course" section. From there, you can create a new section to course link, which will create a new course with the same content as the section you selected. You can also view the list of existing section to course links, refresh them, and delete them.
 
+Relative due dates (optional)
+=============================
+
+If you want to configure relative deadlines in your course, follow these steps:
+
+#. Mark a subsection in the newly created course as graded (otherwise, deadlines will be enforced but learners will not see these dates in the LMS).
+#. Enter the number of weeks in the subsection's "Due in" field.
+#. You may also want to adjust the new course's grading policy to change the weight of the section.
```

Comment:
@Kelketek, I can add something like `**Note:** The start date of a newly created course is in the future, so you will likely want to modify it in the "Schedule & Details" section in Studio.` in the README, but this is a universal thing in Open edX. All courses created in Studio have their start date in the future.

Reply:
@Agrendalath I know-- just wanted to make a note of it in case someone expected the newly generated courses to be available because the source and destination courses have their content published.

Reply:
@Kelketek, sure, added.

Reply:
Thanks!